### ISSUE TITLE
Creating a bundle product in Magento-2.1.4 , while saving a bundle product with single drop-down product option duplicates the same option after saving the product.

### ISSUE THREAD

**Issue by tanmay-mishra831993** [magento/magento2#9565]
Trying to create a bundle product in Magento-2.1.4 but while saving a bundle product with single drop-down product option duplicates the same option after saving the product.

### Steps to reproduce
1.  Create bundle product from magento admin
2.  Assign simple product in Bundle Items tab
3. Save

### Expected result

1. It should save the product with proper configuration, with all options without duplication

### Actual result

1. [Screenshot
![qmd7y](https://cloud.githubusercontent.com/assets/26037453/25852260/657b4058-34e6-11e7-994e-c2ac20550eba.png)
, logs]

The product after save gets its product option gets duplicated if you delete the extra options it will generate more options.

**Comment by tanmay-mishra831993** [comment]
I found the reason of this conflict, in my case this was an issue because of an extension (Magento 2 SEO Extension), that I used for sitemap page generation.

I deleted the above module and removed the database modification done by it and bundle product started working properly again.

Hope it may help other who are facing the same problem due to this same scenario.

Thanks !!! Happy Coding :)
